src/games/strategy/engine/data/export/RulesAttachmentExporter.java

```diff
@@ -134,10 +134,8 @@ private String mTurnsHandler(final Field field, final IAttachment attachment) {
     try {
       oValue = field.get(attachment);
     } catch (final IllegalArgumentException e) {
-      // TODO Auto-generated catch block
```

Comment:
We shoudl do a ClientLogger.logError instead of printing a stack trace. Also capture in the error message the name of the field an attachment would be useful as well.